jackson-support-core/src/main/java/com/messagemedia/framework/jackson/core/valuewithnull/ValueWithNull.java

```diff
@@ -115,6 +116,14 @@ public T get() {
         return value;
     }
 
+    /**
+     *
+     * @return Optionally wrapped underlying value.
+     */
+    public Optional<T> getOpt() {
```

Comment:
Can we call it toOptional or asOptional?

Otherwise: thanks for the PR, i wanted to put one up, but I got distracted...